apps/ledger-live-desktop/tests/models/SwapPage.ts

```diff
@@ -99,6 +99,21 @@ export class SwapPage {
     await this.destinationCurrencyDropdown.click();
   }
 
+  async waitForCurrenciesToExist() {
+    await this.page.waitForTimeout(500);
```

Comment:
What is this 500?
- Waiting for a call....then I assume this logic is not secure.
- Waiting for a transition, can we use a global variable that makes this logic secure if we apply a change in the transition.

Reply:
Agreed, not super. So will take another look at this.

Was caused by a slight delay in the calling of the /to endpoint.

Reply:
Happy if we mock those call in the unit tests.